packages/storage/src/implementation/constants.ts

```diff
@@ -18,15 +18,17 @@
  * @fileoverview Constants used in the Firebase Storage library.
  */
 
+export const domain: string = "firebasestorage.googleapis.com";
```

Comment:
Just want to point out that constant variable names should look like CONSTANT_VALUES_LIKE_THIS per our style guide, but they are all in camelCases in this file.

Would you be able to update them ?

Reply:
Done